parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -198,9 +198,9 @@ write_files:
   permissions: "0644"
   owner: root
   content: |
-    {{if HasDataDir}}
+    {{if HasDataDir -}}
```

Comment:
curious why this is different than L168?

Reply:
good callout, I thought one originally needed extra trim but they are the same. will fix.